install/kubernetes/values.yaml

```diff
@@ -178,7 +178,23 @@ export:
   filenames:
   - tetragon.log
 
+  # stdout specific exporter settings
   stdout:
+    # should the manifest command be overriden?
+    enabledCommand: true
+
+    # should the manifest args be overridden?
+    enabledArgs: true
```

Comment:
These aren't necessary. commandOverride and argsOverride should work the same as they do everywhere else (i.e. empty -> no override).

Reply:
thanks for the very prompt review @willfindlay. the export container I am trying to plug in to replace the stdout one comes with its own default `CMD` and `ARGS`, so there is no need to override it at the manifest level. So, there are 3 states that need 2 flags to control it:
1. Is command override needed at the manifest level? 
2. If the command override is required, is it the default `hubble-export-stdout` or something else.

The same logic applies to `args`. As it is stated in the helm template, the `args` are currently pointing at export directory, but in my case the manifest doesn't require `args` at all. I am passing all the config in the `configMap`.

Hope it makes sense and explains the need for introducing two more flags.

Alternatively, the command and args can be made defaults for the bubble stdout exporter container and omitted altogether. In this case, the `enabledCommand` and `enabledArgs` flags won't be needed.

Reply:
Done. Updated values.yaml with more ingo and  reverted defaults to `true`.